src/components/Card.js

```diff
@@ -36,17 +56,24 @@ export default class Card {
       .cloneNode(true);
   }
 
+  // private method becomes public
+  // for use in handler .then()
   _handleLikeIcon() {
+    // api code goes in the handler in index.js
+    this._handleLikeClick(this);
+
     this._cardElement
       .querySelector(".card__like-button")
       .classList.toggle("card__like-button_active");
```

Comment:
You need to **toggle likes** only in a `then` block after a successful response from the server because if there is an error, the user will not know about that and will see the toggled heart color.